agents/unix/conf/base/conf.c

```diff
@@ -4085,9 +4085,11 @@ mcast_link_addr_list(unsigned int gid, const char *oid,
         return 0;
     }
 #else
+/* It should be big enough to hold 20-octet IPoIB link-layer address. */
+#define MCAST_LINK_ADDR_LEN_MAX (20 * 3)
```

Comment:
20 -> MAX(ETHER_ADDR_LEN, 20)

since we should not lose information that it could be Ethernet address there as well